cli/command/context/list.go

```diff
@@ -76,7 +76,7 @@ func runList(dockerCli command.Cli, opts *listOptions) error {
 	if err := format(dockerCli, opts, contexts); err != nil {
 		return err
 	}
-	if os.Getenv(client.EnvOverrideHost) != "" {
+	if _, present := os.LookupEnv(client.EnvOverrideHost); present {
```

Comment:
I haven't looked closely at the other places where this is checked, but wondering if instead of checking if the env-var is found, we should just take the "simple" approach and treat "empty" and "unset" as equivalent everywhere. 

When scripting, it's not uncommon to have a variable set to an empty string (to "reset it"), and I _think_ it would be ok for us to ignore in that case.